core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -288,6 +288,16 @@ private[chisel3] object ir {
     readUnderWrite: fir.ReadUnderWrite.Value)
       extends Definition
 
+  case class FirrtlMemory(
```

Comment:
Thoughts on naming this `DefMem` and renaming `DefMem` to `CDefMem` for name alignment? Or not that important?

Reply:
I don't think it's important, fine as is IMO